### PR TITLE
Update README and rename method

### PR DESCRIPTION
**Pull request by ChienDevIT**
I corrected it so that it is consistent between README.md and functions. Also correcting error status code 404 in the check function is evaluated as "error" rather than "valid response".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1269,7 +1269,7 @@ ghpr.removeReviewRequests(['user1', 'user2'], callback); //pull request
 
 ### Create release (POST /repos/pksunkara/releases)
 ```js
-ghrepo.release({
+ghrepo.createRelease({
```
Comment (pksunkara) [inline README.md]: https://github.com/pksunkara/octonode/blob/master/src/octonode/repo.coffee#L130

File: README.md [hunk 2]
```diff
@@ -697,7 +697,7 @@ ghrepo.prs(callback); //array of pull requests
 #### Create a pull request (POST /repos/pksunkara/hub/pulls)
 
 ```js
-ghrepo.pr({
+ghrepo.createPr({
```
Comment (pksunkara) [inline README.md]: https://github.com/pksunkara/octonode/blob/master/src/octonode/repo.coffee#L465

**Comment by pksunkara** [comment]
> Also correcting error status code 404 in the check function is evaluated as "error" rather than "valid response".

And that is the way to do API correctly. Any 4xx is error.

**Comment by ChienDevIT** [comment]
> All the current stuff is correct. Your PR just unnecessarily changes the names of things.

I think adding "create" prefixes before POST methods. As you mentioned, it can confuse users with querying a measuring object.

**Comment by ChienDevIT** [comment]
> > All the current stuff is correct. Your PR just unnecessarily changes the names of things.
> 
> I think adding "create" prefixes before POST methods. As you mentioned, it can confuse users with querying a measuring object.

https://developer.github.com/v3/users/followers/#check-if-you-are-following-a-user

That's Github's statement about HTTP status code. Github uses code 204 to represent finding a relationship between two objects (such as A tracking B). And use 404 to show that you can't find it. Therefore, I think 404 should not be evaluated as an HTTP error but should evaluate and handle it as a result of not finding a relationship. If evaluating it as an error, see the code below:

```js
ghme.following('marak', (err, followed) => {
	if(err) return console.error(err);
	console.info("Following marak: " + followed);
});
```

As you say, if you don't track it, it will return err and followed as null. Is that wrong? So I used the algorithm again to accept the 404 status code as a result.
https://github.com/ChienDevIT/octonode/commit/74aa67a91a91c379509f79e5064c626b9844d314

**Pull request closed without merging**